stable/script

```diff
@@ -0,0 +1,8 @@
+SRC="$1"
+DST="$2"
+
+(
+cp -a $SRC/sign/dss $DST/share/
```

Comment:
Barf.

Reply:
All that in the name of backward-compatibility... Will go away in `v3`.